### PR TITLE
Provide eclipse 4.6.3

### PR DESCRIPTION
**Pull request by alarcher**
Redistributable according to license EDLv1.
Verified rule of 3 "gmake publish", and "touch Makefile; gmake publish".

### CODE REVIEW

File: components/developer/eclipse/Makefile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+COMPONENT_VERSION=      4.6.3
+COMPONENT_SUMMARY=      Eclipse IDE for Java, C/C++, Python, JavaScript and PHP
+COMPONENT_PROJECT_URL=  https://eclipse.org/
+COMPONENT_FMRI=         developer/eclipse
+COMPONENT_CLASSIFICATION=Development/Editors
+COMPONENT_SRC=          $(COMPONENT_NAME)
+COMPONENT_ARCHIVE=      $(COMPONENT_NAME)-$(COMPONENT_VERSION).zip
+COMPONENT_ARCHIVE_DATE= 201703010400
+COMPONENT_ARCHIVE_URL= \
+  http://mirrors.nic.cz/eclipse/eclipse/downloads/drops4/R-$(COMPONENT_VERSION)-$(COMPONENT_ARCHIVE_DATE)/eclipse-platform-$(COMPONENT_VERSION)-solaris-gtk-x86_32.zip
```
Comment (pyhalov) [inline components/developer/eclipse/Makefile]: do we want redistribute binaries or perhaps we could build it?

Comment (alarcher) [reply components/developer/eclipse/Makefile]: I think redistributing binaries is good for now, I just patcheed the .ini file to circumvent the GTK issue. Distributions like Arch Linux distribute the binaries.
We may need to update other components (ant?) before we can build it.

Comment (pyhalov) [reply components/developer/eclipse/Makefile]: While building source wasn't found at specified URL...

Comment (pyhalov) [reply components/developer/eclipse/Makefile]: There is one issue, why Arch can have more luck, redistributing binaries: while they are Linux, we are not Solaris...

Comment (alarcher) [reply components/developer/eclipse/Makefile]: The mirror looks weird today...

Comment (alarcher) [reply components/developer/eclipse/Makefile]: Interesting... pkgdepend does not work the same on my workstation and my desktop....

**Comment by alarcher** [comment]
Updated.

**Comment by pyhalov** [comment]
Dumps for me now and then.... 
```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0xfffffd7ffce62c43, pid=2634, tid=0x0000000000000002
#
# JRE version: OpenJDK Runtime Environment (8.0_131) (build 1.8.0_131-internal-oi-1.8.131-b00)
# Java VM: OpenJDK 64-Bit Server VM (25.131-b00 mixed mode solaris-amd64 compressed oops)
# Problematic frame:
# C  [libgobject-2.0.so.0.4600.2+0x42c43]  g_type_class_ref+0x33
#
# Core dump written. Default location: /export/home/alp/core or core.2634
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

```

**Comment by pyhalov** [comment]
http://buildzone.oi-build.r61.net/eclipse-crash/hs_err_pid2634.log

**Comment by alarcher** [comment]
Interesting, I have not seen this ... Been working with 4.6.x for a few months on 3 different machines. I wonder what differs.

**Comment by jimklimov** [comment]
@alarcher, Is JDK the same? e.g. OI OpenJDK vs Sun/Oracle tarball?

**Comment by alarcher** [comment]
@jimklimov OI OpenJDK is used.

**Pull request closed without merging**